### ISSUE TITLE
Normal 1.18 Knockback

### ISSUE THREAD

**Issue by DasLixou** [Minestom/Minestom#944]
Hello!

I use this for knockback:
```kotlin
        eventHandler.addListener(EntityAttackEvent::class.java) { event ->
            val target = event.target
            val attacker = event.entity
            target.takeKnockback(0.4f, Math.sin(attacker.position.yaw * (Math.PI / 180)), -Math.cos(attacker.position.yaw * (Math.PI / 180)))
        }
```
math is same as written in Minestom Docs to function takeKnockback.

But it doesnt fell vanilla. It's to few knockback but when hitting twice, its too much and so on. Onetime my player just "jumped" 200 blocks high

**Comment by DasLixou** [comment]
**EDIT** Could be fixed with #943

**Comment by Bloepiloepi** [comment]
Yep, should be fixed by that, but there are some other issues with entities so it probably won't be merged yet
